tldw_Server_API/app/main.py

```
@@ -713,204 +749,253 @@ async def lifespan(app: FastAPI):
-        if config and config.has_section('Chat-Module'):
-            chat_config = dict(config.items('Chat-Module'))
+    # Heavy initializations: helpers and shared runner to avoid duplication
+    async def _init_mcp_server(*, deferred: bool) -> None:
+        try:
+            from tldw_Server_API.app.core.MCP_unified import get_mcp_server
+            mcp_server = get_mcp_server()
+            if not deferred:
+                logger.info("App Startup: Initializing MCP Unified server...")
+            await mcp_server.initialize()
+            logger.info(("Deferred startup: " if deferred else "App Startup: ") + "MCP Unified server initialized successfully")
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)